src/smart-components/role/roles.js

```diff
@@ -44,6 +45,12 @@ const Roles = () => {
 
   useEffect(() => {
     fetchData({ ...pagination, name: filterValue });
+    window.insights.chrome.getUserPermissions().then(
```

Comment:
Can you gate this behind app permissions? `window.insights.chrome.getUserPermissions(cost-management)`